Launcher/Classes/PlistOperations.swift

```diff
@@ -9,6 +9,23 @@ class PlistOperations {
         dictionaryKey = key
         let documentDirectory = NSSearchPathForDirectoriesInDomains(.cachesDirectory, .userDomainMask, true)[0] as String
         plistPath = documentDirectory.appending(path)
+        
+        if let integratedSettingsPath = Bundle.main.path(forResource: "CalabashLauncherSettings", ofType: "plist"), !fileManager.fileExists(atPath: plistPath){
+            self.copyFile(atPath: integratedSettingsPath, toPath: plistPath)
+        }
+    }
+    
+    func copyFile(atPath: String, toPath: String) {
```

Comment:
An option would be to make this method throwing, so that the caller can handle the error instead of catching it here and handling it generally (where information will get lost about the context when the error occurred).

Reply:
Would be nice if you can show me how to do and what are the benefits of this :)